addons/website_blog/views/website_blog_templates.xml

```diff
@@ -408,7 +408,7 @@
         <ul class="nav nav-pills nav-stacked">
             <t t-foreach="tags" t-as="nav_tag">
             <li t-att-class="nav_tag.id in active_tag_ids and 'active' or None" style="display: inline-block;">
-                <a t-attf-href="#{blog_url(tag=tags_list(active_tag_ids, nav_tag.id))}">
+                <a t-att-rel="'nofollow' if active_tag_ids else None" t-attf-href="#{blog_url(tag=tags_list(active_tag_ids, nav_tag.id))}">
```

Comment:
` t-att-rel="active_tag_ids and 'nofollow'" ` :)

Reply:
AFAIK a `False` value would print `rel="False"`, but `None` would print no tag at all. Isn't it that way?

Reply:
A long time ago, it was true

Now, Falsy -> no att
'' --> att=''

Reply:
so, it is no attribute at all, or is it (in this case) `rel=""` (I'm curious).

Reply:
`<h2 t-att-data-none='None' t-att-data-jke="False and 'ok'" t-att-data-empty='""' t-att-data-false='False' t-att-data-zero='0' class="o_default_snippet_text">Your Banner Title</h2>`

==>

`<h2 class="o_default_snippet_text" data-empty="">Your Banner Title</h2>`

demo here: http://603203-10-0-3862f6-all.runbot27.odoo.com/?db=603203-10-0-3862f6-all#

Reply:
Interesting! It renders:

    <h2 class="o_default_snippet_text" data-empty="">Your Banner Title</h2>

Wow, `t-att-data-zero='0'` disappears too! :open_mouth: 

Thanks for the test!

Reply:
Prefer to use the attf syntax when you want to support the 0 value


`<h2 t-att-data-falsy="False" t-attf-data-fFalsy="#{False}" t-attf-data-fzero="#{0}" t-att-data-zero='0'>`

==>

`<h2 data-fFalsy="" data-fzero="0">`
 